.storybook/config.js

```diff
@@ -57,6 +58,66 @@ export const wInfo = text =>
   withInfo({ inline: true, source: true, styles: wInfoStyle, text: text });
 
 addDecorator(withKnobs);
+
+const GlobalStyle = createGlobalStyle`
+  html, body{
+    font-weight: ${defaultTheme.typography.fontWeight};
+    font-family: ${defaultTheme.typography.fontFamily};
+    font-size: ${defaultTheme.typography.fontSize};
+    color: ${defaultTheme.colors.primary}
+    line-height: ${defaultTheme.typography.lineHeight};
+  }
+
+  pre, code{
+    white-space: pre;
+    white-space: pre-wrap;
+    white-space: -moz-pre-wrap;
+    white-space: -o-pre-wrap;
+  }
+
+  pre{
+    font-family: ${defaultTheme.typography.pre.fontFamily};
+    font-size: ${defaultTheme.typography.pre.fontSize};
+    line-height: ${defaultTheme.typography.pre.lineHeight};
+  }
+
+  code{
+    font-family: ${defaultTheme.typography.code.fontFamily};
+    font-size: ${defaultTheme.typography.code.fontSize};
+    line-height: ${defaultTheme.typography.code.lineHeight};
+  }
+
+  #content code{
+    display: block;
+    padding: 1.5em 2em;
+    border: 1px solid 
+  }
+
+  .tippy-popper[x-placement^=bottom] .tippy-arrow{
+    border-bottom: 8px solid ${defaultTheme.colors.white};
+  }
+
+  .tippy-popper[x-placement^=right] .tippy-arrow{
+    border-right: 8px solid ${defaultTheme.colors.white};
+  }
+
+  .tippy-popper[x-placement^=left] .tippy-arrow{
+    border-left: 8px solid ${defaultTheme.colors.white};
+  }
+
+  .tippy-popper[x-placement^=top] .tippy-arrow{
+    border-top: 8px solid ${defaultTheme.colors.white};
+  }
+
+`;
+
+addDecorator(story => (
+  <React.Fragment>
+    <GlobalStyle />
+    {story()}
+  </React.Fragment>
+));
```

Comment:
Can those styles be applied within the component so that step isn't necessary?

Reply:
@ben-dalton For some, Yes

However, for global fallbacks and defaults like the font-family and base font - it will prove to be repetitive to do so. 

On the case of `tippy` since we need the the arrow to be styled on both the Popover and Tooltip, I thought it'd better suited as a GlobalStyle.

I've decoupled the GlobalStyle from Storybook and added it to the default theme file from which it can be imported and added to the root or any component